src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/ResourceMonitoringOptions.cs

```diff
@@ -29,6 +24,10 @@ public class ResourceMonitoringOptions
     /// <value>
     /// The default value is 5 seconds.
     /// </value>
+    /// <remarks>
+    /// This value represents the total amount of time for which the resource monitor tracks utilization
+    /// information for the system.
+    /// </remarks>
     [TimeSpan(MinimumSamplingWindow, MaximumSamplingWindow)]
     public TimeSpan CollectionWindow { get; set; } = DefaultCollectionWindow;
```

Comment:
From my experience those three time values are often misunderstood by users; from what I've seen they often get invlaid utilization value (NaN), while changing them. I would consider providing less control or extend our docs on how to set it up and what to expect by changing each of the time span value here.